src/banded_global_aligner.cpp

```diff
@@ -0,0 +1,1698 @@
+//
+//  banded_global_aligner.cpp
+//  
+//
+//  Created by Jordan Eizenga on 7/25/16.
```

Comment:
Yes exactly. Git can keep track of who made things when, but the why isn't
captured.

On Thu, Aug 18, 2016 at 8:44 AM, Erik Garrison <notifications@github.com>
wrote:

> In src/banded_global_aligner.cpp
> <https://github.com/vgteam/vg/pull/444#discussion_r75333753>:
>
> > @@ -0,0 +1,1698 @@
> > +//
> > +//  banded_global_aligner.cpp
> > +//
> > +//
> > +//  Created by Jordan Eizenga on 7/25/16.
>
> @adamnovak <https://github.com/adamnovak> I think the objective is to
> enable a faster alignment mode when we have a good idea of the overall
> structure of the alignments.
>
> I guess you mean we should add commentary into the source. That would be
> helpful.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vgteam/vg/pull/444/files/c322adfcc65aaf83c9b54594f7dd51ac1388462d#r75333753>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE0_X_JBMYQc0JHgae3xO-496ee0iUi1ks5qhH35gaJpZM4Jiv8l>
> .
>